include/toBeSorted/event_manager.h

```diff
@@ -35,12 +35,17 @@ class EventManager {
         return strequals(getCurrentEventName(), name);
     }
 
-private:
+    // Something like isActorInEvent maybe?
+    static bool FUN_800a0570(dAcBase_c *actor) {};
+    static bool FUN_800a0ba0() {};
+
     /* 0x000 */ u8 _000[0x084 - 0x000];
     /* 0x084 */ Event mCurrentEvent;
     /* 0x0C4 */ u8 _0C4[0x184 - 0x0C4];
     /* 0x184 */ int mState;
     /* 0x18C */ u8 _18C[0x19C - 0x18C];
+
+private:
```

Comment:
If a private member needs to be accessed, try using a Getter first, less cleanup in the future